Source/Particles/MultiParticleContainer.cpp

```diff
@@ -1601,7 +1601,7 @@ void MultiParticleContainer::CheckQEDProductSpecies()
 
 void MultiParticleContainer::ScrapeParticles (const amrex::Vector<const amrex::MultiFab*>& distance_to_eb)
 {
-#if AMREX_USE_EB
+#ifdef AMREX_USE_EB
```

Comment:
@MKieburtz This should rather go in a mini-PR to upstream development since it is not really related to this PR at all.

Reply:
I can't compile this without it:
```
 /home/mkieburtz/Documents/WarpX/Source/Particles/MultiParticleContainer.cpp:1604:17: error: #if with no expression
   1604 | #if AMREX_USE_EB
        |                 ^
```

Reply:
It looks like the whole bottom section of that file is inside a `#ifdef WARPX_QED` block. On circleCI we use `WarpX_QED=OFF` (which saves on compile time and overall complexity) so we should be fine without the fix for now. But it is worth notifying the WarpX developers of that issue (maybe on Slack) since I'm sure they will be happy to fix another bug in the code.